### PR TITLE
[record-minmax-conversion-test] Add output path option

### PR DESCRIPTION
**Pull request by mhs4670go**
This commit adds output path option to script

ONE-DCO-1.0-Signed-off-by: seongwoo <mhs4670go@naver.com>

### CODE REVIEW

File: compiler/record-minmax-conversion-test/gen_h5_random_inputs.py [hunk 1]
```diff
@@ -8,24 +8,27 @@
 # This script generates a pack of random input data (.h5) expected by the input tflite model
 #
 # Basic usage:
-#   gen_h5_inputs.py --model <path/to/tflite/model> --num_data <number/of/data>
-#   ex: gen_h5_inputs.py --model add.tflite --num_data 3
+#   gen_h5_inputs.py --model <path/to/tflite/model> --num_data <number/of/data> --out <path/to/output/data>
+#   ex: gen_h5_inputs.py --model add.tflite --num_data 3 --output add.tflite.input.h5
```
Comment (seanshpark) [inline compiler/record-minmax-conversion-test/gen_h5_random_inputs.py:12]: this is `--output` , option is `--out`

**Comment by mhs4670go** [comment]
@seanshpark @jinevening PTAL:)

**Comment by seanshpark** [comment]
please check debug test failure

**Comment by mhs4670go** [comment]
@seanshpark @jinevening PTAL:)

**Pull request merged**